Resources/input/fsh/ISiKDiagnose.fsh

```diff
@@ -10,6 +10,8 @@ Description: "Dieses Profil ermöglicht die Nutzung von Diagnosen in ISiK Szenar
   * ^slicing.rules = #open
 * extension contains $condition-related named related 0..1 MS
 * clinicalStatus MS
+  * ^definition = "Einschränkung der übergreifenden MS-Definition: Verfügt ein bestätigungsrelevantes System nicht über die Datenstruktur zur Hinterlegung des Status einer Diagnose, so MUSS dieses System die Information NICHT abbilden. Das System MUSS jedoch den Status kodieren in der Diagnose, sofern die Information verfügbar ist."
```

Comment:
suggestion: dieses Muster (Verpflichtung zur Umsetzung wenn die Information vorhanden ist und Befreiung von der Umsetzung wenn die Information nicht im System geführt wird) wird sich anscheinend häufiger wiederholen. Lohnt es sich dafür einen "übergreifenden Begriff" einzuführen, bspw. "Should Support" statt "Must Support" und diesen Begriff bei allen ähnlichen Situation zu referenzieren?

Reply:
Guter Hinweis. Das wäre etwas für den PR hier https://github.com/gematik/spec-ISiK-Basismodul/pull/412

Reply:
Mit der Formulierung "Should Support" würde ich aber aufpassen... ich denke es ist ein BEDINGTES Must Support.: ein Hersteller MUSS XY, aber nur wenn er in der Lage dazu ist. 
D.h. Vielleicht "CONDITIONAL Must Support" ?!